ws.c

```diff
@@ -37,6 +38,11 @@
 #endif
 
 #define WS_UUID "258EAFA5-E914-47DA-95CA-C5AB0DC85B11"
+/*
+ * We limit the size of received WS frames to 10 MiB,
+ * as a DoS prevention measure.
+ */
+static const size_t WS_MAX_RECV_FRAME_SZ = 10485760;
```

Comment:
How about making it configurable? This can be made as a follow up PR

Reply:
I would love to do this as a follow-up PR yes. I also agree making it configurable is a must.